generator/src/generators/space_age.cr

```diff
@@ -29,7 +30,7 @@ class SpaceAgeTestCase < ExerciseTestCase
   )
 
   def workload
-    "SpaceAge.#{property}(\"#{input.planet}\", #{input.seconds}).should eq(#{expected})"
+    "Space::Age.from_seconds(#{input.seconds}).#{property}_on_#{input.planet.downcase}.should be_close(#{expected}, 0.01)"
```

Comment:
I do like this more OOP approach. I think that's a good example to have.

Reply:
Thanks :slightly_smiling_face: